test/functional/host/errata/recalculate_test.rb

```diff
@@ -0,0 +1,46 @@
+require File.join(File.dirname(__FILE__), '../../test_helper')
+
+require 'hammer_cli_katello/content_view_puppet_module'
+
+describe 'recalculate errata' do
+  include ForemanTaskHelpers
+
+  before do
+    @cmd = %w(host errata recalculate)
+  end
+
+  let(:host_id) { 1 }
+  let(:task_id) { '5' }
+  let(:response) do
+    {
+      'id' => task_id,
+      'state' => 'stopped'
+    }
+  end
+
+  it "recalculates errata for a host" do
+    params = ["--host-id=#{host_id}"]
+
+    ex = api_expects(:host_errata, :applicability, 'Host errata recalculate') do |par|
+      par['host_id'] == host_id
+    end
```

Comment:
I'd prefer the `with_params` syntax. It provides better output (shows all API params expected/actual) when the tests fail. It looks like this:

```rb
api_expects(:host_errata, :applicability, 'Host errata recalculate')
  .with_params('host_id' => host_id)
  .returns(response)
```

The same for the other test.

Reply:
I copied this test from the other test in the folder ([apply_test.rb](https://github.com/Katello/hammer-cli-katello/blob/master/test/functional/host/errata/apply_test.rb)).
But i can change this.